mbtools/models.py

```diff
@@ -214,6 +214,13 @@ def location(self):
         lesson_name = self.etree.xpath("../../name")[0].text
```

Comment:
Leaving this here for lack of a better spot: It looks like in [a previous PR](https://github.com/openstax/raise-mbtools/pull/73) we accidentally committed a `toc.csv` to this repo. Can we remove it in this one?